src/toast/tests/ops_sim_tod_conviqt.py

```diff
@@ -574,154 +574,199 @@ def test_sim_TEB_conviqt(self):
             np.testing.assert_almost_equal(cl_out, cl_outw, decimal=2)
 
         return
-    
+
     def test_sim_hwp(self):
         if not ops.conviqt.available():
             print("libconviqt not available, skipping tests")
             return
 
 
-        data = create_satellite_data(
-            self.comm, obs_time=120 * u.min, pixel_per_process=2
+        data_wo_hwp = create_satellite_data(
+            self.comm, obs_time=120 * u.min, pixel_per_process=2, hwp_rpm=None,
         )
-        self.make_beam_file_dict(data)
-        
-        
+        data_w_hwp = create_satellite_data(
+            self.comm, obs_time=120 * u.min, pixel_per_process=2, hwp_rpm=100,
+        )
+        self.make_beam_file_dict(data_wo_hwp)
+        self.make_beam_file_dict(data_w_hwp)
```

Comment:
Thanks a lot @keskitalo ! i think we're  ready to merge this PR. 
Just a minor comment. 
I don't think this extra-call  `make_beam_file_dict` is needed. If look at its   definition in https://github.com/hpc4cmb/toast/blob/daa4d7e06c3a1f4e949dc6fab71ae43241ebea8e/src/toast/tests/ops_sim_tod_conviqt.py#L215 
essentially what it does is to override the dictionary `beam_file_dict`. Perhaps i can work out a fix to this with something like: 

```
def make_beam_file_dict(self, data): 

        fname2 = self.fname_beam.replace(".fits", "_bottom.fits") 
        beam_file_dict = {}
        for det in data.obs[0].local_detectors:
            if det[-1] == "A":
                beam_file_dict[det] = self.fname_beam
            else:
                beam_file_dict[det] = fname2

        return beam_file_dict 

``` 
please let me know if that makes the test routine cleaner .